src/components/App/App.js

```diff
@@ -11,17 +11,22 @@ function App() {
   const [player1, setPlayer1] = useState({})
   const [player2, setPlayer2] = useState({})
   const [winner, setWinner] = useState("")
+  const [dropDownInput, setDropDownInput] = useState("all")
 
   function selectCharacter(newCharacter){
     setCharacter(newCharacter)
   }
 
   function selectPlayer1(input){
+    setCharacter({})
     setPlayer1(input)
+    getCharacters()
   }
-
+  
   function selectPlayer2(input){
+    setCharacter({})
     setPlayer2(input)
+    getCharacters()
```

Comment:
Could there be a way to combine these two to make it more dynamic and more DRY? This is fine for now but we should keep this in mind when refactoring